Makefile

```diff
@@ -37,7 +37,7 @@ test: test-cli test-cgi test-py test-webservice
 
 # Removes the compiled files
 clean:
-	$(RM) bin/newtonraphson.exe src/py/newtonraphsonpy.*.so apache2/cgi-bin/newtonraphson
+	$(RM) bin/newtonraphson.exe src/py/newtonraphsonpy.*.so apache2/cgi-bin/newtonraphson src/js/newtonraphsonwasm.js  src/js/newtonraphsonwasm.wasm
 
 start-redis:
 	docker run --rm -d -p 6379:6379 --name some-redis redis
```

Comment:
in order to use container port mapping one needs to be super user or adjust the system. This should be clear as well.

Reply:
Done in edd3287e68c47c7b0bb744d379b893edba29947f